### PR TITLE
Added min_last_updated_datetime to FetchAccountBalanceAsync

### PR DESCRIPTION
**Pull request by sistemd**
This is necessary for us to be able to solve https://github.com/ln-zap/zap-middleware/issues/2732. It's been tested via the UI flow, and should be correct, but please test it yourself as well.

### CODE REVIEW

File: src/Plaid/SerializableContent.cs [hunk 1]
```diff
@@ -20,7 +20,7 @@ public virtual string ToJson()
 		{
 			return JsonConvert.SerializeObject(this, new JsonSerializerSettings()
 			{
-				DateFormatString = "yyyy-MM-dd",
+				DateFormatString = "yyyy-MM-ddTHH:mm:ssZ",
```
Comment (mrfelton) [inline src/Plaid/SerializableContent.cs]: Why is this change necessary? Could this have unintended side effects for other parts of the code?

Comment (sistemd) [reply src/Plaid/SerializableContent.cs]: That's true, it could have unintended side effects. https://github.com/LN-Zap/Plaid.NET/pull/6/commits/e391485cc83fbf4c4074b4f5aa66a9af65e018ce
It's necessary because the docs say they expect an ISO-8601 date BTW https://plaid.com/docs/api/products/#accounts-balance-get-request-min-last-updated-datetime_options

**Review by mrfelton — APPROVED** [review]
